source/vibe/http/websockets.d

```diff
@@ -697,108 +727,234 @@ final class IncomingWebSocketMessage : InputStream {
 	}
 }
 
+/// Magic string defined by the RFC for challenging the server during upgrade
+private static immutable s_webSocketGuid = "258EAFA5-E914-47DA-95CA-C5AB0DC85B11";
```

Comment:
`static immutable` does: It means the variable is a CT-known constant that will end up in the data section. It is quite useful for arrays, for example:

```D
static immutable data = [1, 2, 3, 4];

void process (immutable int[]) @nogc;

void main () @nogc
{
    process(data);
}
```

If one used `enum` here, `main` couldn't be `@nogc`.